tests/default.nix

```
@@ -5,22 +5,23 @@
   pkgsUnfree,
 }:
 let
-  fetchTests = import ./fetch-tests.nix;
+  fetchTests = import ./fetch-tests.nix { inherit lib pkgs helpers; };
   test-derivation = import ../lib/tests.nix { inherit pkgs lib; };
   inherit (test-derivation) mkTestDerivationFromNixvimModule;
 
   moduleToTest =
-    name: module:
+    file: name: module:
     mkTestDerivationFromNixvimModule {
-      inherit name module;
+      inherit name;
+      module = {
+        _file = file;
```

```suggestion
        _file = "${file}:${name}";
```

This (hack) would cause the test-case name to also be shown in the filename, e.g.
```
error: The option `foo' does not exist. Definition values:
- In `/nix/store/ss06gh6ig3nk0pvh1a7n5x1a3g47x91j-test-sources/examples.nix:simple-plugin': ""
```